src/steps/app-engine/index.ts

```diff
@@ -40,10 +40,7 @@ import {
   createAppEngineVersionEntity,
 } from './converters';
 import { getCloudStorageBucketKey } from '../storage/converters';
-import {
-  CLOUD_STORAGE_BUCKET_ENTITY_TYPE,
-  STEP_CLOUD_STORAGE_BUCKETS,
-} from '../storage';
+import { StorageEntitiesSpec, StorageStepsSpec } from '../storage';
```

Comment:
Can you remove the `export * from './constants'` line from `storage/index.ts`? Exporting from `index.ts` can quickly lead to circular dependencies, so it's much better to always import from `constants` and to stop the practice of `export * from './constants'`.

```suggestion
import { StorageEntitiesSpec, StorageStepsSpec } from '../storage/constants';
```

Reply:
Updated in commit [f007ff0](https://github.com/JupiterOne/graph-google-cloud/pull/556/commits/f007ff0e369a05753dd3522e6461ba097f0deb5d) :)